scripts/genomeNexusInternalApiClientConfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "apiPackage":"org.genome_nexus.client",
+  "modelPackage":"org.genome_nexus.client",
+  "groupId":"org.genome_nexus.client",
+  "artifactId":"genomeNexusInternalApiClient",
+  "artifactName":"Genome Nexus Internal API Client",
+  "artifactVersion":"1.1.1-SNAPSHOT"
```

Comment:
manually synced with the current genome nexus master branch version.

@inodb Do you think it is easy to automate this?

Reply:
@onursumer I think it's fine to manually run the generator script every now and then to update the client. It's not that often we'll have to update the client if all goes well. Just some readme of how to do it would be nice

Reply:
i believe jitpack overrides this version with the commit id or tag, so it prolly doesn't matter what we feel in here